.circleci/config.yml

```diff
@@ -718,6 +718,17 @@ jobs:
       - *persist_to_workspace_step
       - *save_cache_step
 
+  algoliasearch:
+    docker:
+      - *test_runner
+    resource_class: *resource_class
+    steps:
+      - checkout
+      - *restore_cache_step
+      - run: tox -e 'algoliasearch_contrib-{py27,py34,py35,py36}-algoliasearch' --result-json /tmp/algoliasearch.results
```

Comment:
Julien just made some changes to how we run test suites via tox, these got merged in on Friday.

Example of the new script: `run: scripts/run-tox-scenario '^jinja2_contrib-'`